extensions/opc/src/fetchopc.cpp

```diff
@@ -93,14 +93,12 @@ namespace processors {
 
     translatedNodeIDs_.clear();  // Path might has changed during restart
 
-    BaseOPCProcessor::onSchedule(context, factory);
-
-    if(!configOK_) {
-      return;
+    try {
+      BaseOPCProcessor::onSchedule(context, factory);
+    } catch (const Exception &e) {
+      throw e;
     }
```

Comment:
What's the intention behind these lines? We catch `minifi::Exception`s then rethrow them without doing anything.